iron-query-params.html

```diff
@@ -56,8 +56,13 @@
       if (this._dontReact) {
         return;
       }
-      this.paramsString = this._encodeParams(this.paramsObject)
-          .replace(/%3F/g, '?').replace(/%2F/g, '/').replace(/'/g, '%27');
+      let _encParams = this._encodeParams(this.paramsObject)
```

Comment:
You are right, `paramsObj` could be an empty object. Mostly I'm suggesting that if we protect against a falsy `this.paramsString` in the `paramsObjectChanged` method, then `this.paramsString` will not be overwritten incorrectly and the `this.paramsObject` will remain an empty object. These are just my suggestions, seems to be working for my simple use cases, and would be simple.